server/static/gen/explorer/app_css.738c8aed.js

```diff
@@ -0,0 +1,104 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
why are we commenting on this? Does it have some effect?

Reply:
I cant find the file, but it looks like a generated build file.